src/components/TodoListFooter.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import "./TodoListFooter.css";
+
+class TodoListFooter extends React.Component {
+  creatFilter = caption => {
```

Comment:
typo

Reply:
I could also rename to `renderFilter` since it is used like helper for rendering